packages/client3/pages/artist/dashboard/pending-pools-artist.tsx

```diff
@@ -38,25 +41,44 @@ function PendingPoolArtist() {
       walletAddress: account || "",
       filters: {
         statusType: [Pool_Status_Type.Approved, Pool_Status_Type.InReview],
-        hasTransactionHash: false,
+        hasPoolAddress: false,
       },
     },
   });
 
+  const pendingPools = data?.pendingPools ?? [];
+
   const goldfinchFactory = useContract(
     "GoldfinchFactory",
     CONTRACT_ADDRESSES.GoldFinchFactory
   );
 
-  const pendingPools = data?.pendingPools ?? [];
-
   const onContractSubmit = async (pool: typeof pendingPools[0]) => {
     if (!goldfinchFactory) {
       console.error("Goldfinch factory couldn't be initialized");
       return;
     }
-    const receipt = await createPool(goldfinchFactory, pool.goalAmount);
-    await updatePoolTransactionHash(pool.id, receipt);
+
+    const borrowerContract = await createBorrowerContract(
+      goldfinchFactory,
+      pool.walletAddress.toLowerCase()
+    );
```

Comment:
Just to clarify, Is a new borrower contract created every time we create a pool?

Reply:
Yes a new contract will be created for each pool that is created.